src/pyscipopt/scip.pyx

```diff
@@ -1657,6 +1657,46 @@ cdef class Model:
         PY_SCIP_CALL(SCIPupdateNodeLowerbound(self._scip, node.scip_node, lb))
 
     # node methods
+    def getBestChild(self):
+        """gets the best child of the focus node w.r.t. the node selection strategy."""
+        cdef SCIP_NODE* _node
+
+        _node = SCIPgetBestChild(self._scip)
+        node = Node.create(_node)
+        return node
```

Comment:
Thanks! could you rewrite it in a single line, please? Like
`return Node.create(SCIPgetBestChild(self._scip))`